src/VisualizationTests/HelixWatch3DViewModelTests.cs

```diff
@@ -258,7 +258,7 @@ public void Node_Labels_Render()
             OpenVisualizationTest("Labels.dyn");
 
             // check all the nodes and connectors are loaded
-            Assert.AreEqual(2, model.CurrentWorkspace.Nodes.Count());
+            Assert.AreEqual(3, model.CurrentWorkspace.Nodes.Count());
```

Comment:
Since I added a watch node to this Labels.dyn, I had to update the node count in this test